incubator/twitter/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "kubeless-python-simple",
+  "version": "0.0.1",
+  "description": "This example demonstrates how to setup a simple Python function with Kubeless",
+  "dependencies": {
+    "serverless-kubeless": "^0.1.4"
```

Comment:
ah good point. I am currently testing node.js applications, so I thought this was node.js as well.